docs/README.md

```diff
@@ -1 +1,86 @@
 # Checkout Shipping
+
+This app has the `ShippingSummary` and `ShippingForm` components used in the shipping step of the checkout.
+
+![Shipping summary component](./images/shipping_summary.png)
+![Shipping form component](./images/form_postal_code.png)
+
+## Usage
+
+This app is used by [`checkout-step-group`](https://github.com/vtex-apps/checkout-step-group) and shouldn't be used individually. Should you find the need to use it, you need to:
+
+1. Add it as a dependency in your `manifest.json`:
+```json
+{
+  "dependencies": {
+    "vtex.checkout-shipping": "0.x"
+  }
+}
+```
+
+2. Import the components in your code:
+
+```jsx
+import React from 'react'
+import { ShippingPreview, ShippingForm } from 'vtex.checkout-shipping'
+
+const Form = ({ isPreviewMode = false }) => {
+  if (isPreviewMode) {
+    return (
+      <ShippingPreview />
+    )
+  }
+
+  return (
+    // no need to pass in any props, as the component
+    // uses all information provided by the `order-manager` app.
+    <ShippingForm />
+  )
+}
+```
+
+Note that you need to have `OrderFormProvider` from `vtex.order-manager` app somewhere above this
```

Comment:
can we change this to `vtex.order-shipping`? Order manager is just a transient dependency